scripts/Muon/GUI/Common/fitting_widgets/basic_fitting/fit_function_options_view.py

```diff
@@ -12,6 +12,7 @@
 from Muon.GUI.Common.utilities import table_utils
 
 from qtpy.QtWidgets import QWidget
+from qtpy.QtCore import Qt
```

Comment:
Thanks for changing this, can you also put this import above the `qtpy.QtWidgets` import?